lib/copy_peste/core/core_state.rb

```diff
@@ -7,10 +7,14 @@ class CoreState
       attr_accessor :conf
       attr_accessor :events_to_command
       attr_accessor :analysisModOpts
+      attr_accessor :graphic_mod
```

Comment:
wouldn't `attr_reader` be better for both (if ot all) here?
I'm not sure those variables should be modified by any other object than `self`.